internal/clientconn/conninfo/conn_info.go

```diff
@@ -53,6 +65,27 @@ func (connInfo *ConnInfo) SetAuth(username, password string) {
 	connInfo.password = password
 }
 
+// Cursor returns the cursor value stored.
+func (connInfo *ConnInfo) Cursor(collection string) iterator.Interface[int, any] {
+	return connInfo.cursor[collection]
+}
+
+// SetCursor stores the cursor value.
```

Comment:
Maybe it worth to mention that "collection" string should contain both db and collection names not to cause a conflict between the same collections in different dbs.

Reply:
Renamed it to `key` and added a comment about usage.